ncm-fstab/src/main/perl/fstab.pm

```diff
@@ -110,7 +141,7 @@ sub remount_everything
     while (my $f = <$fstab>) {
     	my $mount = $self->mount_from_entry($f);
     	$mount && $mount ne 'swap' or next;
-    	LC::File::makedir ($mount);
+    	LC::Check::directory ($mount);
```

Comment:
@ned21 ncm-fstab itself  seem to work safe under NoAction mode, using CAF or LC::Check. But is depends on ncm-lib-blockdevices.. I don't know if someone has checked this for NoAction..